src/bin/utils.ts

```diff
@@ -155,11 +157,58 @@ async function retryAuth<T>(
   }
 }
 
+const envVariables = {
```

Comment:
All of these functions and properties should be placed into `bin/options.ts` or `bin/parsers.ts`. The `bin/parsers.ts` is a general "marshalling" function for processing arguments and environment variables and relevant parameters.

Reply:
This includes `envVariables`, `getByFile`, `getPassword`, `getRecoveryCode`.

Reply:
Moved them over.